src/handlers.js

```diff
@@ -30,6 +29,22 @@ actionHandlers['!delete'] = {
     }
 };
 
+// =======================================
+// Command: !amazon
+// Description: will display the standard amazon alert with no sound (as not to distract the old man)
+// =======================================
+actionHandlers['!amazon'] = {
+    security: (context, textContent) => {
+        return context.mod || (context["badges-raw"] != null && context["badges-raw"].startsWith("broadcaster"))
+    },
+    handle: (context, textContent) => {
+        var text = "gaming.amazon.com"
```

Comment:
Not a big deal but these can all be `const` since they are not mutated, as far as I can see 😄